pkg/kubelet/config/config_test.go

```diff
@@ -62,7 +63,14 @@ func CreateValidPod(name, namespace, source string) *api.Pod {
 		Spec: api.PodSpec{
 			RestartPolicy: api.RestartPolicyAlways,
 			DNSPolicy:     api.DNSClusterFirst,
-			Containers:    []api.Container{{Name: "ctr", Image: "image", ImagePullPolicy: "IfNotPresent"}},
+			Containers: []api.Container{
+				{
+					Name:            "ctr",
+					Image:           "image",
+					ImagePullPolicy: "IfNotPresent",
+					SecurityContext: securitycontext.ValidSecurityContextWithContainerDefaults(),
```

Comment:
The defaulting takes place in the Container method.  The unit tests that were updated are hand crafting a pod to do their testing.  It might help to have a test utility that returns a valid container that can be manipulated if folks think it would be useful, the `securitycontext.ValidSecurityContextWithContainerDefaults()` method was a first stab.